src/Testcontainers.Azurite/AzuriteConfiguration.cs

```diff
@@ -0,0 +1,213 @@
+namespace Testcontainers.Azurite;
+
+/// <inheritdoc cref="ContainerConfiguration" />
+[PublicAPI]
+public sealed class AzuriteConfiguration : ContainerConfiguration
+{
+  /// <summary>
+  ///   Initializes a new instance of the <see cref="AzuriteConfiguration" /> class.
+  /// </summary>
+  /// <param name="azuriteServices">A value indicating which Azurite service is turned on.</param>
+  /// <param name="blobPort">The Azurite container blob port.</param>
+  /// <param name="blobPortBinding">The host blob port to bind.</param>
+  /// <param name="queuePort">The Azurite container queue port.</param>
+  /// <param name="queuePortBinding">The host queue port to bind.</param>
+  /// <param name="tablePort">The Azurite container table port.</param>
+  /// <param name="tablePortBinding">The host table port to bind.</param>
+  /// <param name="workspaceLocation">The Azurite workspace directory path.</param>
```

Comment:
Yes `WorkspaceLocation` and `WorkspaceLocationBinding` are same as `BlobPort` and `BlobPortBinding` cases. I will update them based on our decision. Because there is one location then it is easier situation than with ports.